sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -173,6 +173,7 @@ def get_query_response(self, request: Request):
     def updateState(self, txns, isCommitted=False):
         for txn in txns:
             self._update_state_with_single_txn(txn, is_committed=isCommitted)
+        super().updateState(txns, isCommitted=True)
```

Comment:
Why `isCommitted=True` here? Please pass the correct value `super().updateState(txns, isCommitted=isCommitted)`